### DISCUSSION TITLE
How is the Geist documentation website build?

### DISCUSSION THREAD

**Discussion by jordihoven** [zeit-ui/react#790] [Q&A] [answered]
I am looking to build a component library for our development team and would love to display components in a similar way as how geist shows it's components (with states, sizes and code snippets).

Now, I'm wondering how the documentation website is build. Does it use something like storybooks?

Would love to know. Thanks

**Comment by unix** [comment, marked as answer]
We didn't use Storybook, we built it entirely in code 🤣 , which is in the repository's `pages` folder.

**Reply by jordihoven** [reply]
Makes sense. I found the 'pages' folder. All MDX files. Cool!
